packages/foreman/nodejs-bootstrap-sass/nodejs-bootstrap-sass.spec

```
@@ -0,0 +1,50 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name bootstrap-sass
+
+Name: %{?scl_prefix}nodejs-bootstrap-sass
+Version: 3.4.3
+Release: 1%{?dist}
+Summary: bootstrap-sass is a Sass-powered version of Bootstrap 3, ready to drop right into your Sass powered applications
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/twbs/bootstrap-sass#readme
+Source0: https://registry.npmjs.org/bootstrap-sass/-/bootstrap-sass-%{version}.tgz
+%if 0%{?!scl:1}
+BuildRequires: nodejs-packaging
+%endif
+BuildArch: noarch
+ExclusiveArch: %{nodejs_arches} noarch
+Provides: %{?scl_prefix}npm(%{npm_name}) = %{version}
+
+%description
+%{summary}
+
+%prep
+%setup -q -n package
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr assets %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr eyeglass-exports.js %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr package.json %{buildroot}%{nodejs_sitelib}/%{npm_name}
+
+%nodejs_symlink_deps
+
+%check
+%{nodejs_symlink_deps} --check
+
+%files
+%{nodejs_sitelib}/%{npm_name}
+%license LICENSE
+%doc CHANGELOG.md
+%doc README.md
+
+%changelog
+* Mon Jun 16 2025 MariaAga <mariaaga@redhat.com> 3.4.3-1
+- Update to 3.4.3
+
+* Mon Jun 16 2025 MariaAga <mariaaga@redhat.com> 3.4.3-1
```

```suggestion
* Mon Jun 16 2025 MariaAga <mariaaga@redhat.com> 3.4.3-1
```

last nitpick, I promise! (and will commit it myself!)